democracy/migrations/0030_migrate_translatable_fields.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.10 on 2016-12-08 12:00
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations
+from django.core.exceptions import ObjectDoesNotExist
+
+IMPORT_LANGUAGE_CODE = 'fi'  # Assume that the existing content is in Finnish
```

Comment:
It is possible, but if there is only a few non Finnish Hearings, I would recommend against since the detection may be incorrect or even fail, leading us with probably more Hearing to fix manually.